base/Dockerfile

```diff
@@ -81,3 +81,6 @@ RUN echo '. /etc/skel/.welcome' >> /home/st2/.bashrc
 # Used by all stackstorm services
 VOLUME ["/etc/st2"]
 WORKDIR /opt/stackstorm
+
+# Remove openssh server keys
+RUN rm -f /etc/ssh/ssh_host*
```

Comment:
Please move the step just after `st2` package installation step so we remove these keys at earlier stage.

Reply:
Thanks @armab done!